docs/user-quick-start.md

```diff
@@ -171,6 +171,17 @@ enough resources:
    For more info see
    [MinIO Client Quickstart](https://min.io/docs/minio/linux/reference/minio-mc.html#quickstart)
 
+1. Install `kustomize` tool
+
+   ```
+   curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash
+   sudo install kustomize /usr/local/bin
+   rm kustomize
+   ```
+
+   For more info see
+   [kustomize install](https://kubectl.docs.kubernetes.io/installation/kustomize/)
```

Comment:
Thanks! but can you match the way we document this in test/README.md?
https://github.com/RamenDR/ramen/commit/0926e409aacc1105786f40da2f1dee98dc059647

Reply:
Should I maybe swap the one in README for the one I am adding?
this one is a little more elaborate, similarly to others in the quick start doc

Reply:
Yes, let's apply your version to test/README.md. The downside is that now we need to update this when kustomize changes the instructions, but we plan to automated this part soon.